portfolio.html

```diff
@@ -176,5 +176,85 @@ <h1>About me</h1>
         </ul>
       </div>
     </section>
+    <section id="contact">
+      <div>
+        <h2>
+          I'm always interested in hearing about new projects, so if you'd like
```

Comment:
- [ ] [OPTIONAL] This paragraph should be a `p` tag. The `p` tag defines a paragraph of text